.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- '6.10.2'
+- '8.11.3'
```

Comment:
Sorry, this is different from the `nodejs_version` in `appveyor.yml` below; intentional?

Reply:
Wait, hasn't it got updated? How come? It shows updated on my local.

Reply:
Done!